ansible/roles/update/handlers/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+
+- name: reboot
```

Comment:
So basically when a reboot is required, it will do it immediately, right? This is fine by me, but on the forum it may cause trouble.
Do you know if it's possible for it to schedule a reboot for some time in the night? Or would that require too much work. If yes, then let's do it this way

Reply:
It will call reboot when a reboot and the `reboot_enabled` variable is set true, by default `reboot_enabled` variable is not set.

Reply:
Ok. thx